library/farmosUtil/farmosUtil.transaction.js

```diff
@@ -28,6 +28,71 @@
  * If an operation was never attempted (and thus also not undone) the attribute for that operation will be `undefined`.
  *
  * @category Utilities
+ * 
```

Comment:
It looks like there are lines here that duplicate what is above.  Be sure there is just one copy of them in the resulting file.  Keep the ones above and remove the ones below.